cdptools/bin/clone_db.py

```diff
@@ -25,9 +25,13 @@ def __parse(self):
             prog="clone_prod",
             description="Clone prod database to target database."
         )
-        p.add_argument("project_id", help="Project id of source database.", default="cdp-seattle")
+
         p.add_argument("target_credentials", type=Path, help="Credentials of target database.")
```

Comment:
I know you didn't touch this but to make it in the same style as the source, can you `--` as well.